sdk/trace-shaded-deps/src/test/java/io/opentelemetry/sdk/trace/internal/SunMiscProhibitedSecurityManagerTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.trace.internal;
+
+import java.security.AccessControlException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class SunMiscProhibitedSecurityManagerTest {
+
+  @Test
+  public void test_prohibits_sun_misc_package_access() {
```

Comment:
The `test` prefix is not helpful. Let's replace it with the method under test `checkPackageAccess`. Additionally, this project uses CamelCase in method names. How about `checkPackageAccess_ProhibitsSunMisc`. 

This advice applies to the other tests as well.

Reply:
fixed